### ISSUE TITLE
Issues with pop-up tools in Krita when tile windows is enabled

### ISSUE THREAD

**Issue by senntore** [pop-os/shell#1316]
**(1) Issue/Bug Description:**
When tiling mode is enabled globally, pop-up  tools like **Color Selector** doesn't work as expected, they either appear on screen for a second or does not appear at all, when they're supposed to disappear once I toggle it using a shortcut or by clicking somewhere else.

Right now I can't get these tool to show properly:
- **show color selector**
- show color history
- show common colors
- show minimal shade selector

**(2) Steps to reproduce (if you know):**
1. Disable Tiling Windows, if it's enabled using Super + Y
2. Open Krita
3. Try using various tools listed above(if you don't know or have shortcuts for them, you can use Ctrl+Enter to search them)
4. Note the current behavior
5. Enable Tiling Windows
6. Repeat Steps 2 & 3
7. See the issue.

**Screencast**:

https://user-images.githubusercontent.com/58325574/149346917-37444635-355e-4842-8e92-03ed3ffaa978.mp4

**(3) Expected behavior:**
Those tools mentioned above should only disappear once user toggle it using a shortcut or click somewhere else.

**(4) Distribution (run `cat /etc/os-release`):**
Pop OS 21.10 

**(5) Gnome Shell version:**
gnome-shell:
  Installed: 40.5-1ubuntu2pop0~1633630918~21.10~91d3049
  Candidate: 40.5-1ubuntu2pop0~1633630918~21.10~91d3049
  Version table:
 *** 40.5-1ubuntu2pop0~1633630918~21.10~91d3049 1001
       1001 http://apt.pop-os.org/release impish/main amd64 Packages
        100 /var/lib/dpkg/status
     40.5-1ubuntu2 500
        500 http://us.archive.ubuntu.com/ubuntu impish/main amd64 Packages

**(6) Pop Shell version (run `apt policy pop-shell` or provide the latest commit if building locally):**

```
pop-shell:
  Installed: 1.1.0~1641595320~21.10~c3d93f8
  Candidate: 1.1.0~1641595320~21.10~c3d93f8
  Version table:
 *** 1.1.0~1641595320~21.10~c3d93f8 1001
       1001 http://apt.pop-os.org/release impish/main amd64 Packages
       1001 http://apt.pop-os.org/release impish/main i386 Packages
        100 /var/lib/dpkg/status

```
**Krita version**: 5.0.2

**(10) Other Notes:**
What I've tried:
- Adding Krita to floating window exception or using Super + G has no effect & these tools only show properly after disabling tiling mode with Super + Y.
- Testing the issue in older version of Krita(v5), it was working fine before but now shows this issue.

The issue started happening since yesterday(or maybe the day before)

**Comment by mmstick** [comment]
Likely already fixed by #1314

**Comment by senntore** [comment]
Oh thanks! I saw that pr previously but forgot that it could be related to this one. I'll reopen the issue if it persist after that update.
